shared_infra/sqs_redrive/run.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+import logging
+import os
+
+import boto3
+import daiquiri
+
+daiquiri.setup(level=logging.INFO)
+logger = daiquiri.getLogger(__name__)
+
+
+def send_message(sqs_client, msg, queue_url):
+    sqs_client.send_message(
+        QueueUrl=queue_url,
+        MessageBody=msg
+    )
+
+
+def delete_message(sqs_client, msg, queue_url):
+    sqs_client.delete_message(
+        QueueUrl=queue_url,
+        ReceiptHandle=msg['ReceiptHandle']
+    )
+
+
+def main(sqs_client):
+    source_queue_url = os.environ["SQS_SOURCE_URL"]
+    target_queue_url = os.environ["SQS_TARGET_URL"]
+    while True:
+        response = sqs_client.receive_message(
+            QueueUrl=source_queue_url
+        )
+
+        if "Messages" not in response:
```

Comment:
Easier-to-Ask-For-Permission (EAFP) is generally considered more Pythonic than Look-Before-You-Leap. So it’s slightly more idiomatic to write:

```python
try:
    for msg in response['Messages']:
        ...
except KeyError:
    break
```

But this approach is fine, and I don’t care strongly about which one you use.